Lib/fontTools/feaLib/parser.py

```diff
@@ -404,7 +404,9 @@ def parse_glyph_pattern_(self, vertical):
             return ([], prefix, [None] * len(prefix), values, [], hasMarks)
         else:
             assert not any(values[:len(prefix)]), values
-            values = values[len(prefix):][:len(glyphs)]
+            format1 = values[len(prefix):][:len(glyphs)]
+            format2 = values[len(prefix) + len(glyphs):][:len(suffix)]
```

Comment:
i got confused here by the relative priority of `+` and `:` operators. Some parentheses may help readability maybe

```suggestion
            format2 = values[(len(prefix) + len(glyphs)):][:len(suffix)]
```